recipes/ffmpeg/all/conanfile.py

```diff
@@ -642,11 +642,12 @@ def opt_append_disable_if_set(args, what, v):
         ranlib = buildenv_vars.get("RANLIB")
         if ranlib:
             args.append(f"--ranlib={unix_path(self, ranlib)}")
-        # for some reason pkgconf from conan can't find .pc files on Linux in the context of ffmpeg configure...
```

Comment:
"some reason"
the `pkgconf` from the Conan package is not able to locate `.pc` files for system dependencies, e.g. in `/usr/lib/aarch64-linux-gnu/libfoo.pc`, because it is hardcoded to look at `/lib/pkgconfig` and `/share/pkgconfig` - on Ubuntu there's no system .pc files in those directories. 

it works when `pkgconf` is used to locate conan-generated .pc files, and on platforms where PKG_CONFIG_PATH is defined in the shell environment (like msys2) - but won't work on Linux. 

We should always recommend that on Linux (and any other similar OS that already has a pkg-config), that the system pkg-config is used, and let build systems fall back to locating a prefixed pkg-config when crossbuilding - and this should be the default behaviour without requiring additional configuration

Reply:
Also, besides xorg, none of the packages really need to be `/system`, really:
- `vaapi/system`: https://github.com/valgur/conan-center-index/blob/dev/recipes/libva/all/conanfile.py
- `vdpau/system`: https://github.com/valgur/conan-center-index/blob/dev/recipes/vdpau/all/conanfile.py